main.html

```diff
@@ -270,59 +395,99 @@ <h2 class="accesibility__header"> Making millions of rides more <br> accessible
         </div>
         <div class="business">
             <figure class="business__headers">
-                <img class = "business__image-source" src ="lyft-logo.png" alt=""/>
-                <span class="business-pic__header"> business </span> 
+                <img class = "business__image-source" src="lyft-logo.png" alt="lyft-logo"/>
+                <span class="business-pic__header">business</span> 
             </figure>
-            <span class="business__header"> We help move <br> employees </span>
+            <span class="business__header">We help move <br> employees</span>
             <p class="business__text"> 
-                    From affordable transportation solutions to <br>
-                    efforthless expensing, we'll help drive your <br>
-                    business forward by <br>
-                    getting your people where they need to go. <br>
+                    From affordable transportation solutions to 
+                    efforthless expensing, we'll help drive your 
+                    business forward by
+                    getting your people where they need to go. 
             </p>
-            <button class="business__button-source"> Learn more
-            </button>
-            <div class="business__partnerimage">
-                <img class="business__partnerimage-source" src ="Airbnb-Logo.png" alt =""/>
+            <button class="business__button-source">Learn more</button>
+            <div class="business__partner-image">
+                <img class="business__partner-image-source" src="Airbnb-Logo.png" alt ="partners-airbnb"/>
+                <img class="business__partner-image-source-delta" src="delta.png" alt ="partners-delta"/>
+                <img class="business__partner-image-source-ascension" src="ascension.png" alt ="partners-ascension"/>
             </div>
             <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="business__arrow" viewBox="0 0 16 16">
                 <path fill-rule="evenodd" d="M1 8a7 7 0 1 0 14 0A7 7 0 0 0 1 8zm15 0A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/>
             </svg>
         </div>
         <div class="type">
```

Comment:
I'd avoid generic classnames like that, and use more explicit names, it's totally fine to spend more time trying to come up with a cool name than writing a code after that

Reply:
+